pontoon/base/views.py

```diff
@@ -353,31 +352,25 @@ def contributor(request, email, template='user.html'):
     return render(request, template, data)
 
 
-def contributors(request, template='users.html'):
+def contributors(request, period='alltime', template='users.html'):
     """Top contributors view."""
     log.debug("Top contributors view.")
 
-    translations = (
-        Translation.objects
-        .exclude(user=None)
-        .exclude(string=F('entity__string'))
-        .exclude(string=F('entity__string_plural'))
-    )
-
-    users = (
-        User.objects
-        .filter(translation__in=translations).distinct()
-        .exclude(email__in=settings.EXCLUDE)
-        .annotate(translation_count=Count('translation'))
-        .exclude(translation_count=0)
-        .order_by('-translation_count')[:100]
-    )
-
-    for user in users:
-        user.translations = translations.filter(user=user)
+    now = timezone.now()
+    start_date = None
+    end_date = utils.aware_datetime(now.year, now.month, 1)
+
+    try:
+        period = int(request.GET['period'])
+    except (KeyError, ValueError):
+        period = None
+
+    if period:
```

Comment:
You can put this if statement in the try block and then just have the except block be a `pass` to make it slightly simpler to read.